lambdas/service/app.py

```diff
@@ -359,9 +357,21 @@ def get_summary(entity_type=None):
     es_td = EsTd()
     # Get the response back
     logger.info("Creating the API response")
-    response = es_td.transform_summary(filters=filters, entity_type=entity_type)
+    response_files = es_td.transform_summary(filters=filters, entity_type="files")
+    response_specimens = es_td.transform_summary(filters=filters, entity_type="specimens")
+
+    unified_response = {}
+    for field in ("totalFileSize", "fileTypeSummaries", "fileCount"):
+        unified_response[field] = response_files[field]
+    for field in ("projectCount", "organCount", "donorCount",
+                  "labCount", "totalCellCount", "organSummaries", "specimenCount"):
+        unified_response[field] = response_specimens[field]
+    unified_keys = unified_response.keys()
+    if len(unified_keys) != len(response_files.keys()) and len(unified_keys) != len(response_specimens.keys()):
```

Comment:
This should be an assertion, meaning it should use the `assert` statement.

Also, `len(d) == len(d.keys())` for every d with `instanceof(d, dict)`.